assets/blocks/lesson-actions/complete-lesson-block/index.js

```diff
@@ -12,7 +12,7 @@ export default createButtonBlockType( {
 		parent: [ 'sensei-lms/lesson-actions' ],
 		title: __( 'Complete lesson', 'sensei-lms' ),
 		description: __(
-			'Enable an enrolled user to mark the lesson as complete.',
+			'Enable an enrolled user to mark the lesson as complete. The button is displayed when the user is enrolled and there is no required quiz linked to the lesson.',
```

Comment:
@donnapep I updated the descriptions of the lesson buttons since I think that we should describe to the user when each of the buttons is going to be displayed in the frontend. Let me know of any suggestions.